apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/Diagnosis.java

```diff
@@ -125,14 +122,14 @@ public void setLabels(DiagnosisLabel label) {
    * Enumerates the various labels/tags that are used to distinguish between the various diagnoses
    * in a claim.
    */
-  static enum DiagnosisLabel {
-    /** Note: display text matches {@link ExDiagnosistype#PRINCIPAL}. */
+  public static enum DiagnosisLabel {
+    /** Note: display text matches ExDiagnosistype#PRINCIPAL. */
```

Comment:
I'm thinking we should stay consistent and utilize JS Doc {@link 

this comment applies to all comments which have had this tag removed.

Reply:
Some of the Java doc comments have been modified to be independent of HAPI FHIR data model version to be available as a common component for both BFD v1 and BFD v2.